datahub-web-react/src/app/shared/tags/AddTagsTermsModal.tsx

```diff
@@ -351,7 +375,15 @@ export default function EditTagTermsModal({
             })
             .catch((e) => {
                 message.destroy();
-                message.error({ content: `Failed to remove: \n ${e.message || ''}`, duration: 3 });
+                if (resources.length > 1 && getGraphqlErrorCode(e) === 403) {
+                    message.error({
+                        content:
+                            'Your bulk edit selection included entities that you do not own. The bulk edit being performed will not be saved.',
```

Comment:
You don't necessarily need to "own" an asset to be able to make changes. You just need to have the correct Metadata Privilege, which can include privileges like "Edit Tags", "Edit Glossary Terms" and more.